tests/test_kernel_module.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from pytest_container import container_from_pytest_param
+from pytest_container import DerivedContainer
+from pytest_container.container import ContainerData
+
+from bci_tester.data import KERNEL_MODULE_CONTAINER
+
+
+_DRBD_VERSION = "9.2.5"
+
+
+DRBD_CONTAINER = DerivedContainer(
+    base=container_from_pytest_param(KERNEL_MODULE_CONTAINER),
+    containerfile=rf"""WORKDIR /src/
+RUN zypper -n in coccinelle tar
+
+RUN set -euxo pipefail; \
+    curl -L -o drbd.tar.gz https://pkg.linbit.com//downloads/drbd/9/drbd-{_DRBD_VERSION}.tar.gz; \
+    tar xvzf drbd.tar.gz; rm drbd.tar.gz; cd drbd-{_DRBD_VERSION}; \
+    make -C /usr/src/linux-obj/x86_64/default modules M="$(pwd)/drbd" SPAAS=false
```

Comment:
this is not gonna work on non-x86_64, but the test is not restricted to be run only on x86_64